x-pack/plugins/security_solution/common/endpoint/utils/commands.ts

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
if you endup keeping these (see other comments below) I think they should go into `common/endpoint/service/response_actions` instead, since they are all `actions` specific